src/Generating/CMakeLists.txt

```diff
@@ -54,6 +55,7 @@ target_sources(
 	Prefab.h
 	PrefabPiecePool.h
 	PrefabStructure.h
+        SinglePieceStructuresGen.h
```

Comment:
Use tabs for indent

Reply:
Already fixed and changed the settings in my IDE (drives me crazy)

Reply:
Normal IDEs should be able to pick up our editorconfig file, so no setup should be necessary. Unless you use something exotic.

Reply:
I use CLion and this mostly happens when I use the rename feature - it even recognizes that there is a clang file included